phpBB/includes/ucp/ucp_pm_compose.php

```diff
@@ -751,6 +751,34 @@ function compose_pm($id, $mode, $action, $user_folders = array())
 		$enable_urls 		= (isset($_POST['disable_magic_url'])) ? 0 : 1;
 		$enable_sig			= (!$config['allow_sig'] ||!$config['allow_sig_pm']) ? false : ((isset($_POST['attach_sig'])) ? true : false);
 
+		/**
+		* Modify private message
+		*
+		* @event core.ucp_pm_compose_modify_submit_before
+		* @var	bool	enable_bbcode		Whether or not bbcode is enabled
+		* @var	bool	enable_smilies		Whether or not smilies are enabled
+		* @var	bool	enable_urls			Whether or not urls are enabled
+		* @var	bool	enable_sig			Whether or not signature is enabled
+		* @var	string	subject				PM subject text
+		* @var	object	message_parser		The message parser object
```

Comment:
Have you checked what this was replaced with in 3.2.x? Will probably not be the message parser.